koch.nim

```diff
@@ -428,6 +430,48 @@ proc xtemp(cmd: string) =
   finally:
     copyExe(d / "bin" / "nim_backup".exe, d / "bin" / "nim".exe)
 
+proc runCI(cmd: string) =
+  doAssert cmd.len == 0, cmd # avoid silently ignoring
+  echo "runCI:", cmd
+  # todo: consider replacing ./koch commands below with direct proc call, eg:
+  # exec "./koch boot -d:release" => boot("-d:release")
+  # or, using `quoteShell(getAppFilename())` instead of koch
+
+  when defined(posix): # appveyor didn't run this
+    # todo: pending https://github.com/nim-lang/Nim/issues/7999 ; instead of this hack, use `execCmd` with env
```

Comment:
Remove this 'todo'. Add a ``proc execWithEnv`` to ``kochdocs.nim`` instead, comparable to ``execCleanPath``.